inc/template-tags.php

```diff
@@ -124,9 +124,32 @@ function pingback_header() {
  * Add Google Analytics Tracking to the site.
  */
 add_action(
-	'wp_head', function() {
+	'wp_head',
+	function() {
 		ai_get_template_part( 'template-parts/global/google-analytics' );
-	}, 15
+	},
+	15
+);
+
+/**
+ * Add Google Tag Manager Tracking to the site.
+ */
+add_action(
+	'wp_head',
+	function() {
+		ai_get_template_part( 'template-parts/global/google-tag-manager' );
```

Comment:
You're already hooking `wp_head` with a priority of `15` to add Google Analytics - any reason why you wouldn't want to put this in the same function?